packages/snaps-cli/src/webpack/loaders/wasm.ts

```diff
@@ -95,8 +96,9 @@ const loader: LoaderDefinitionFunction = async function loader(
 
   // Add the WASM import as a dependency so that Webpack will watch it for
   // changes.
+  const path = dirname(this.resourcePath);
   for (const name of Object.keys(imports)) {
-    this.addDependency(name);
+    this.addDependency(resolve(path, name));
```

Comment:
Dependencies must use absolute paths. This solves a warning when using the WASM loader.